src/inc/msquic.h

```diff
@@ -464,6 +464,10 @@ typedef enum QUIC_PERFORMANCE_COUNTERS {
     QUIC_PERF_COUNTER_WORK_OPER_QUEUE_DEPTH,// Current worker operations queued.
     QUIC_PERF_COUNTER_WORK_OPER_QUEUED,     // Total worker operations queued ever.
     QUIC_PERF_COUNTER_WORK_OPER_COMPLETED,  // Total worker operations processed ever.
+    QUIC_PERF_COUNTER_PATH_VALIDATED,       // Total path challenges that succeed ever.
```

Comment:
In QUIC_PARAM_GLOBAL_PERF_COUNTERS, we verify the buffer for this is exactly QUIC_PERF_COUNTER_MAX. So we need to update the logic there to allow both old and new sizes, otherwise its a breaking change.

Reply:
I see the following in there:
```c
        if (*BufferLength < QUIC_PERF_COUNTER_MAX * sizeof(int64_t)) {
            //
            // Copy as many counters will fit completely in the buffer.
            //
            *BufferLength = (*BufferLength / sizeof(int64_t)) * sizeof(int64_t);
        } else {
            *BufferLength = QUIC_PERF_COUNTER_MAX * sizeof(int64_t);
        }
```
This should be fine, right?

Reply:
Yup I misread.